migrations/20220614123448_add_recipe_and_order_token_id.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async (knex) => {
+  await knex.schema.alterTable('recipes', (def) => {
+    def.integer('token_id')
```

Comment:
maybe should be called `latest_token_id`?

Reply:
...or we don't have the column and use `_transactions` tables instead

Reply:
no this column should be here and perhaps we should have 2: `latest_token_id` and `original_token_id`. What do you think?